Userland/Libraries/LibCore/Stream.cpp

```diff
@@ -753,7 +753,7 @@ WrappedAKOutputStream::WrappedAKOutputStream(NonnullOwnPtr<OutputStream> stream)
 
 ErrorOr<Bytes> WrappedAKOutputStream::read(Bytes)
 {
-    VERIFY_NOT_REACHED();
+    return Error::from_errno(EBADF);
```

Comment:
Wait, why EBADF instead of ENOTSUP?

Reply:
Because this matches the behavior of a file descriptor that isn't opened for the respective operation (discussed [here](https://discord.com/channels/830522505605283862/830739873119207426/1052561217836830741)).